### ISSUE TITLE
Add ARIA Roles to README.md

### ISSUE THREAD

**Issue by varjmes** [hoodiehq/hoodie-css#11]
We have a number of examples of HTML use within the README.md
I think we should include the aria roles on the elements shown too, part of our want to be accessible from the start.

Thoughts: @zoepage & @lewiscowper ?

**Comment by zoepage** [comment]
Yes, this is the idea. But we have to do this, after the content is 100% done. So we don't do things twice ;)
